var/spack/repos/builtin/packages/highfive/package.py

```diff
@@ -19,6 +19,7 @@ class Highfive(CMakePackage):
     license("BSL-1.0")
 
     version("develop", branch="master")
+    version("2.10.1", sha256="548569311da2ff2c7bf020eaaed24678c172f778d6a155b7eb11a3e7e32d6647")
```

Comment:
I am not getting this sha256. How was it calculated?

What I'm getting instead:
```suggestion
    version("2.10.1", sha256="60d66ba1315730494470afaf402bb40300a39eb6ef3b9d67263335a236069cce")
```

Reply:
The one's for the zip archive and the other for the tar.gz.

Reply:
Ah. It needs to be the one for the version equivalent of the `url` property.  (See https://spack.readthedocs.io/en/latest/packaging_guide.html#version-urls.)